calico-cloud_versioned_docs/version-18-2/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -82,8 +82,7 @@ You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, G
 - Download the CLI scanner binary onto your CI runner
 - If you are running an ephemeral environment in the pipeline, include the download, and update the executable steps in your pipeline to download the scanner on every execution
 - Create a secret containing the API-Token and API URL and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
```

Comment:
Where is `SECURE_API_TOKEN` from? That seems like a CI platform dependency environment variable name. I think we should probably avoid that here since our instructions are generic / agnostic.

Reply:
Minor: Is there a reason for the "-" in "API-Token"?

Reply:
This was altered text from sysdig doc for our purposes. Hence, the discrepancies.